src/mdbExtensionController.ts

```diff
@@ -487,6 +487,38 @@ export default class MDBExtensionController implements vscode.Disposable {
         return Promise.resolve(true);
       }
     );
+    this.registerCommand(
+      EXTENSION_COMMANDS.MDB_COPY_DOCUMENT_ID_FROM_TREE,
+      async (element: DocumentTreeItem): Promise<boolean> => {
+        const { documentId } = element;
+
+        if (!documentId) {
+          void vscode.window.showWarningMessage('Can\'t copy ID from document that has no ID.');
+          return false;
+        }
+
+        let body = JSON.stringify(documentId);
```

Comment:
What happens here if `documentId` contains non-JS bson datatypes, e.g. `Binary()` or similar? Should this be EJSON instead of JSON?